nengo/tests/conftest.py

```diff
@@ -111,10 +137,13 @@ def pytest_generate_tests(metafunc):
 
 
 def pytest_addoption(parser):
-    parser.addoption('--benchmarks', action='store_true', default=False,
-                     help='Also run benchmarking tests')
-    parser.addoption('--plots', action='store_true', default=False,
-                     help='Also run plotting tests')
+    parser.addoption(
+        '--plots', nargs='?', default=False, const=True,
+        help='Save plots (optional with directory to save them in).')
+    parser.addoption(
+        '--analytics', nargs='?', default=False, const=True,
+        help='Also run analytics tests (optional with directory to save ' +
```

Comment:
I would say "Save analytics" instead of "Also run analytics tests", analogous to `--plots`.